packaging/linux/run_keybase

```diff
@@ -45,7 +45,7 @@ echo Launching keybase service...
 keybase service &>> "$logdir/keybase.service.log" &
 echo Mounting /keybase...
 kbfsfuse -debug -mdserver mdserver.kbfs.keybase.io:443 \
-  -bserver bserver.kbfs.keybase.io:443 /keybase &>> "$logdir/keybase.kbfs.log" &
+  -bserver bserver.kbfs.keybase.io:443 -log-to-file /keybase &>> "/dev/null" &
```

Comment:
No redirection of stderr is needed here. With log-to-file kbfs should not be writing anything to stderr unless:
1) the log file cannot be opened for writing (redirection won't help...)
2) kbfsfuse call syntax is erroneous and the command just exists as command line flag parsing fails

Also are explicit -bserver and -mdserver needed here with production builds?